index.html

```diff
@@ -137,12 +137,32 @@
     </div>
   </script>
 
+  <!-- Project List item -->
+  <script type="text/template" id="project-list-template">
+    <%
+      var classNames = [
+        "project-list",
+        starred ? "starred" : "",
+        featured ? "featured" : "",
+      ].join(" ");
+    %>
+    <div id="<%- id %>" 
+          class="<%- classNames %>" 
+          data-title="<%- title %>">
+      <div class="content">
+        <a href="<%- detailViewLink %>" class="read-more-link"><h2><%- title %></h2></a>
+        <h3><% if (creators) { %> <%- creators %> <% } %><% if (creators && timestamp)  %><% if (timestamp) { %> on <%- timestamp %> <% } %></h3>
+      </div>
+    </div>
+  </script>
+
   <script src="https://code.jquery.com/jquery-2.1.4.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.5.1/lodash.min.js"></script>
   <script src="js/utility.js"></script>
   <script src="js/notifier.js"></script>
   <script src="js/favourites-manager.js"></script>
   <script src="js/project-card.js"></script>
+  <script src="js/project-list.js"></script>
```

Comment:
Could we rename this to `project-list-item.js` as well? And don't forget to change the file name!